context.go

```diff
@@ -130,79 +129,61 @@ var testContextData = struct {
 }{service: make(map[string][]byte, 0)}
 
 // Save takes an identifier and an interface. The interface will be network.Marshaled
-// and saved under a filename based on the identifier. An eventual error will be returned.
-// If contextDataPath is non-empty, the destination is a file: it will be created
-// with rw-r----- permissions (0640). If the file already exists, it will be overwritten.
-//
-// The path to the file is chosen as follows:
-//   Mac: ~/Library/Conode/Services
-//   Other Unix: ~/.local/share/conode
-//   Windows: $HOME$\AppData\Local\Conode
-// If the directory doesn't exist, it will be created using rwxr-x---
-// permissions (0750).
-//
-// The path can be overridden with the environmental variable "CONODE_SERVICE_PATH".
+// and saved in the database under the bucket named after the service name.
+// The database must be created by the server prior to using this function.
 func (c *Context) Save(id string, data interface{}) error {
 	buf, err := network.Marshal(data)
 	if err != nil {
 		return err
 	}
-	fname := c.absFilename(id)
-	if getContextDataPath() == "" {
-		testContextData.Lock()
-		testContextData.service[fname] = buf
-		testContextData.Unlock()
-		return nil
-	}
-
-	return ioutil.WriteFile(fname, buf, 0640)
+	bucketName := ServiceFactory.Name(c.ServiceID())
```

Comment:
You could put `bucketName` in `Context`, then it's available for `Save` and `Load`.